src/zimscraperlib/zim/filesystem.py

```diff
@@ -210,3 +211,86 @@ def make_zim_file(
         raise
     finally:
         zim_file.finish()
+
+
+class BadZimfileDataError(Exception):
```

Comment:
I'd say that _Bad_ prefix for those error is _bad_ 😉 _Incorrect_ seems more appropriate

- `IncorrectZIMPathError`
- `MissingZIMFolderError`
- `NotADirectoryZIMFolderError`
- `NotWritableZIMFolderError`
- `IncorrectZIMFilnameError`

Reply:
sorry for the *bad* namings 😅